src/Symfony/SymfonyRequestParserFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace MPScholten\RequestParser\Symfony;
+
+use MPScholten\RequestParser\RequestParser;
+use MPScholten\RequestParser\RequestParserFactory;
+use Symfony\Component\HttpFoundation\Request;
+
+class SymfonyRequestParserFactory implements RequestParserFactory
```

Comment:
The `Symfony\RequestParser` and `Psr7\RequestParser` classes got renamed to `Symfony\RequestParserFactory` and `Psr7\RequestParserFactory` to avoid a name collision as there is already a `RequestParser` class which is doing something different. There's also a new `RequestParserFactory` interface with `createQueryParser` and `createBodyParser`.